Models/DTOs/Output/ReceiptOutput.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace PSP.Models.DTOs.Output
+{
+    public class ReceiptOutput
```

Comment:
As we discussed, receipts should have related products and services. (or do you want to add this in another PR?)